storybook/src/Storybook/FigmaImagesProxyModel.qml

```diff
@@ -0,0 +1,30 @@
+import QtQuick 2.14
+
+ListModel {
+    /* required */ property FigmaLinksCache figmaLinksCache
+    property alias sourceModel: d.model
+
+    readonly property Instantiator _d: Instantiator {
```

Comment:
Nice trick!

Reply:
it helps me to keep `FigmaPreviewWindow` and nested components like `FigmaImagesGridView` as "standard" as possible. They just consume model. Otherwise I would have to pass callback to them making it the not so generic.